### PR TITLE
Added a new tutorial to Examples section

### PR DESCRIPTION
**Pull request by mameshini**
Added a new pipelines tutorial
Fixed title font to match other sections

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/902)

### CODE REVIEW

**Comment by mameshini** [comment]
Fixes #799

**Comment by joeliedtke** [comment]
/hold

Hold for technical review.

File: content/docs/examples/resources.md [hunk 1]
```diff
@@ -68,6 +68,12 @@ and codelabs that are hosted outside the Kubeflow repositories.
 Run Kubeflow on Red Hat [OpenShift](https://www.openshift.com/).
 {{% /blocks/content-item %}}
 
+{{% blocks/content-item title="Agile Stacks Kubeflow Pipelines Tutorial"
+  url="https://www.agilestacks.com/tutorials/ml-pipelines" %}}
+Run Kubeflow Pipelines tutorials on AWS, GCP, or on-prem hardware using [AgileStacks](https://www.agilestacks.com/).
```
Comment (sarahmaddox) [inline content/docs/examples/resources.md]: s/AgileStacks/Agile Stacks

**Comment by sarahmaddox** [comment]
Thanks @joeliedtke and @mameshini! Just one small typo to fix. When that's done, I'll approve this PR.

/hold cancel

File: content/docs/examples/resources.md (hunk 1 above)
Comment (mameshini) [reply content/docs/examples/resources.md]: Thanks, I fixed typo

**Pull request merged**